lib/game.rb

```diff
@@ -23,14 +23,30 @@ def make_move
     @display.display_board(@board.board)
     @display.show_current_player(@current_player.mark)
     move = @display.ask_for_move
-    @board.player_make_move(@current_player.mark, move)
-    @display.display_board(@board.board)
+    until move_valid?(move)
+      @display.invalid_move_message(move)
+      move = @display.ask_for_move
+    end
+    @board.player_make_move(@current_player.mark, move.to_i)
```

Comment:
Question - think Single Responsibility Principle... What is the responsibility of a Game - perhaps managing the relationship between a board and 2 players... Should it also be responsible for dealing with invalid input?

Reply:
Good point! I think validating the move should be in Board. Thanks, Matt